lib/services/database/comment_upvote_repository_service.dart

```diff
@@ -0,0 +1,24 @@
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/database/comment/comment_data.dart";
+import "package:proxima/models/database/comment/comment_firestore.dart";
+import "package:proxima/models/database/comment/comment_id_firestore.dart";
+import "package:proxima/models/database/post/post_firestore.dart";
+import "package:proxima/models/database/post/post_id_firestore.dart";
+import "package:proxima/services/database/firestore_service.dart";
+import "package:proxima/services/database/upvote_repository_service.dart";
+
+final commentUpvoteRepositoryService = Provider.family<
```

Comment:
I think this variable name should finish with `Provider`.

Reply:
You are right !
Fix in 8f4c0e9ddc06351397b6f00fb0568b119f9f5a14.